src/ios/CDVStatusBar.m

```diff
@@ -132,13 +132,15 @@ - (void)pluginInitialize
 
     // blank scroll view to intercept status bar taps
     self.webView.scrollView.scrollsToTop = NO;
-    UIScrollView *fakeScrollView = [[UIScrollView alloc] initWithFrame:UIScreen.mainScreen.bounds];
+    
+    CGRect bounds = [self.viewController.view.window bounds];
```

Comment:
@jonathanli2: When testing this, I get a (0,0)x(0,0) frame:

    (CGRect) bounds = (origin = (x = 0, y = 0), size = (width = 0, height = 0))

... which means I can't tap the status bar.

I'm testing on iOS 10b8, iPhone 6s and iPad Pro (with and without a launch storyboard). 

Could you share a test project that worked for you so I can see what (if any) differences there are that might cause this?